sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs

```diff
@@ -2359,6 +2376,24 @@ public partial class CheckCosmosDBNameAvailabilityResponse
         public Azure.ResourceManager.CosmosDB.Models.CosmosDBNameUnavailableReason? Reason { get { throw null; } }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ClusterType : System.IEquatable<Azure.ResourceManager.CosmosDB.Models.ClusterType>
```

Comment:
The name is too generic, consider to rename it with a more descriptive and meaningful name.

Reply:
Changed to CassandraClusterType